src/sql/workbench/contrib/notebook/test/browser/notebookActions.test.ts

```diff
@@ -396,11 +396,12 @@ suite('Notebook Actions', function (): void {
 			const e: azdata.nb.IKernelChangedArgs = <azdata.nb.IKernelChangedArgs>{
 				newValue: <azdata.nb.IKernel>{
 					name: 'StandardKernel2'
-				}
+				},
+				nbKernelAlias: ''
```

Comment:
Instead of modifying this test it might be best to add another test with that optional argument not there.